### ISSUE TITLE
I pod from the cocopods instal SDWebimag, but the project after the operation showed 'NSImage + WebCache. H' file not found. Why

### ISSUE THREAD

**Issue by SevenHL** [rs/SDWebImage#1990]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by SevenHL** [comment]
SDWebImage version is 4.0.0 and then I try to comment out from podfile file SDWebImage, pod install, but SDWebImage didn't delete from my project.

**Comment by bpoplauschi** [comment]
@SevenHL those are all issues with CocoaPods and not with our project.
Make sure your podfile contains the platform. Example `platform :ios, '8.0'`

**Comment by SevenHL** [comment]
Very thanks for your reply, I think I have already solved the problem, is due to rely on in the pod 'SDCycleScrollView SDWebImage, cause I can't delete this library to import.
